Assets/Scripts/ViconNexusUnityStream/SubjectDataManager.cs

```diff
@@ -6,12 +6,39 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
-public class DataStreamer : Singleton<DataStreamer>
+public class SubjectDataManager : MonoBehaviour
```

Comment:
I am deeply saddened to learn about the loss of my dear friend, Mr. Singleton, who has been an invaluable support throughout my career as a developer. May Singleton rest in peace and find its new home in the hands of a naive developer, continuing to guide and inspire as it did for me.

Reply:
RIP